util/container/image.go

```diff
@@ -17,8 +17,7 @@ limitations under the License.
 // Package container implements container utility functionality.
 package container
 
-import (
-	//  Import the crypto sha256 algorithm for the docker image parser to work
+import ( //  Import the crypto sha256 algorithm for the docker image parser to work
```

Comment:
Looks like the gci linter has some new opinions about comment placement - it looks worse to me, but I'm fine with merging this as it won't impact in many places.

Reply:
Nice. I prefer the new variant. I think it's the mainstream variant (IIRC)

Reply:
Yeah - the current version is nice. The outdated version above was the result from `make lint-fix` and it is ... not nice